src/include/uapi/ipc/topology.h

```diff
@@ -210,11 +211,12 @@ struct sof_ipc_comp_tone {
 	int32_t ramp_step;
 } __attribute__((packed));
 
-/** \brief Types of processing components */
-enum sof_ipc_process_type {
+/** \brief Types of EFFECT */
```

Comment:
@slawblauciak lets stick with processsing components to align with the name change to process